app/modules/individuals/models.py

```diff
@@ -141,7 +141,7 @@ class Individual(db.Model, HoustonModel, CustomFieldMixin):
 
     # there is a backref'd 'relationship_memberships' list of RelationshipIndividualMember accessible here
     def user_is_owner(self, user):
-        return user is not None and user in self.get_members()
+        return user is not None and user in self.get_owners()
```

Comment:
I have a vague recollection that I found calls to get_members in the unit test code. I may be wrong. If the tests pass, I was wrong.